lib/ecto/adapters/myxql/connection.ex

```diff
@@ -551,6 +551,19 @@ if Code.ensure_loaded?(MyXQL) do
 
     defp expr({:count, _, []}, _sources, _query), do: "count(*)"
 
+    defp expr({:json_extract_path, _, [expr, path]}, sources, query) do
+      path =
+        Enum.map(path, fn
+          binary when is_binary(binary) ->
+            [".", binary]
```

Comment:
Do we need to do some sort of escaping here?

Reply:
Yes, see: https://github.com/elixir-ecto/ecto/pull/3227#discussion_r377126347